pkg/config/config.go

```diff
@@ -59,6 +59,8 @@ type Server struct {
 	HideApplications []string `def:"" desc:"please don't use, this will soon be deprecated"`
 
 	OutOfSpaceThreshold bytesize.ByteSize `def:"512MB" desc:"Threshold value to consider out of space in bytes"`
+
+	RetentionThresholdDays int `def:"3" desc:"data retention threshold in days"`
```

Comment:
We should make this a `time.Duration`. ParseDuration does not support setting the duration in days, so we'll need to implement that. For that we can make a wrapper, similarly to how we have a wrapper for `time.Time` values: https://github.com/pyroscope-io/pyroscope/blob/main/pkg/cli/cli.go#L51-L74